src/main/java/com/parth/StudentManagementMyBatisJwt/services/StudentService.java

```diff
@@ -55,18 +58,19 @@ public StudentDisplayDto addStudent(StudentAdditionDto studentAdditionDto) {
     return studentMapper.convertStudentEntityToStudentDisplayDto(studentEntity);
   }
 
+  @Transactional(readOnly = false, isolation = Isolation.READ_COMMITTED, propagation = Propagation.REQUIRED, rollbackFor = DataFormatException.class)
```

Comment:
Do we need to set all these attributes?

Reply:
in our case no need to see
Code updated